DNSDist.py

```diff
@@ -12,13 +12,14 @@
 
 
 class Console(object):
-    def __init__(self, key=None, host='127.0.0.1', port=5199, have_sodium=HAVE_SODIUM):
+    def __init__(self, key=None, host='127.0.0.1', port=5199, have_sodium=HAVE_SODIUM, merge_nonces=True):
```

Comment:
This behavior is a breaking change for anyone not on 1.2.0+. Since we don't have versioning, the default behavior should be off, not on.

Reply:
You are right! Fixed